### PR TITLE
2432 introduce an option to animate panel and slide background images

### PR DESCRIPTION
**Pull request by MichelaSalvemini**
fixed TFillScale and added ease-in-out to MZoomAnimation

### CODE REVIEW

File: lib/material/measing.flow [hunk 1]
```diff
@@ -698,7 +700,7 @@ MZoomAnimation(
 		],
 		MMEasingAnimation(
 			const(0.3),
-			const(easingFastOutSlowIn),
+			const(CubicBezierEasing(0.42, 0.0, 0.58, 1.0)),
```
Comment (mmissiato) [inline lib/material/measing.flow:703]: Strange this constants in a component.
 @NikitaFil maybe you know better.

File: lib/tropic/tropic2form.flow [hunk 2]
```diff
@@ -1281,7 +1281,9 @@ tropic2Acc(b : Tropic, parent : TParentInfo, sheet : Stylesheet, metricsOnly : b
 					xscale = fdivide(ba2Metrics.width, ba1.metrics.width);
 					yscale = fdivide(ba2Metrics.height, ba1.metrics.height);
 					customScale = extractStruct(style, TFillScale(const(0.0))).scale;
-					scale = fmax3(xscale, yscale, customScale);
+					scale =
```
Comment (mmissiato) [inline lib/tropic/tropic2form.flow:1284]: @NikitaFil please check this too.

Comment (NikitaFil) [reply lib/tropic/tropic2form.flow:1284]: Don't change TFillScale behaviour, it will break existing content

File: lib/material/measing.flow (hunk 1 above)
Comment (NikitaFil) [reply lib/material/measing.flow:703]: It isn't a standard curve, but no objections about custom curve usage

**Review by NikitaFil — CHANGES_REQUESTED** [review]
Are you sure you need MTweak here? Wouldn't it be easier to use just MScale?

**Comment by NikitaFil** [comment]
@discane If you are fine with this new TFillScale behaviour, then I won't insist on changes. New naming / comment may be needed though.

**Pull request merged**